<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPersonalAccount</class>
 <widget class="QMainWindow" name="FormPersonalAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FormPersonalAccount</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="teTextToEncode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblEncode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Encode</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDecode">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Decode</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teTextFromFileWithDecode">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFilePath">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblFilePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FilePath:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEncode">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Encode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGetTextFromFileWithDecode">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Get text with decode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>290</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>btnExit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
